QUESTIONS.md

```diff
@@ -1,5 +1,14 @@
 # Questions about Code, Node, Express, React
 
 What questions can you come up with?
-
+- More on React
+- More CSS please!
+- More HTML!
+- What is front end?
+ 
 ## Cohort Questions
+- When will profolio site be covered
+- Is there a syllabus we can see
+- Is there a way to know what assignments we still need to hand in
+- When will SQL be introduced // been told to use for industry project
```

Comment:
I also can't wait to learn more about mySQL, I think it will be very useful for our industry project and want to compare it to mongoDB.

Reply:
same!